src/params/syntax/async-gen-named-func-expr.template

```diff
@@ -13,7 +13,6 @@ info: |
         7. Let closure be ! AsyncGeneratorFunctionCreate(Normal, FormalParameters,
            AsyncGeneratorBody, funcEnv, strict).
         [...]
-flags: [async]
```

Comment:
It's currently only used for syntax error tests. The real issue is that the template generates tests with the "async" flag, but doesn't call $DONE.